Kinder/Views/Home/Index.cshtml

```diff
@@ -1,2 +1,58 @@
-﻿
+﻿<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="~/lib/bootstrap/dist/css/bootstrap.min.css" />
+    <link rel="stylesheet" href="~/css/mainStyle.css" />
+    <link rel="stylesheet" href="~/css/site.css" />
+</head>
+<body class="mainBackground">
+    <div>
+        <p class="welcome">
+            Welcome to Kinder! Give away. Get karma points. Compete with other people!
+        </p>
+    </div>
+    <div class="rect">
+        <img src="~/Rectangle 3.png" />
+    </div>
+    <div class="rect1">
+        <img src="~/Rectangle 1.png" />
+    </div>
+    <div class="par4">
+        Each year, a lot of variuos trash is being produced. And sometimes, items that appear in our trash container, are not actually a real trash!
+    </div>
+    <div class="par2">
+        Some people just want to free some space from their variuos items, which are not being used by themselves
+        , but can be useful for some people.
+    </div>
+    <div class="rect4">
+        <img src="~/Rectangle 4.png" />
+    </div>
+    <div class="par6">
+        So in order to fight this problem, our team created an application which will help people to give away some items that they dont need anymore,
+        and also to give them not for utilization, but for those ones who really need something.
+    </div>
+    <div class="par3">
+        Give it a try
+    </div>
+    <div class="par5">
+        How does this application work?
+    </div>
+    <div class="par7">
+        This application works on the concept of "swiping" that is being used in some other applications like "Tinder", so thats why it is called Kinder :)
```

Comment:
This application works on the concept of "swiping" that can be seen in some other applications like "Tinder" so the app should feel familiar to anyone that has seen or used such apps.

Reply:
List items that you don't need
People who need them will like them
Choose who will get your item
Earn karma points and compete with other people on the leaderboards!